dev_db/creator.py

```diff
@@ -36,25 +38,23 @@ def get_models(self):
         excluded = self.get_excluded_models()
         all_models = self.get_all_models()
         valid_models = list(full_required)
-        
 
         for m, n in all_models:
             # skip the ones already in full required
             if m in valid_models:
                 continue
-            table = m._meta.db_table
-            # handle the excludes
-            valid = True
-            for e in excluded:
-                if e in table:
-                    valid = False
-            if getattr(m._meta, 'proxy'):
-                valid = False
-            if valid:
-                valid_models.append(m)
-                
+
+            if all(map(lambda x: x not in m._meta.db_table, excluded)) \
+                    and not getattr(m._meta, 'proxy'):
+                try:
+                    m.objects.first()  # trigger potential database errors
+                except Exception as e:
+                    logger.error('%s: %s', type(e).__name__, e)
+                else:
+                    valid_models.append(m)
```

Comment:
This is what is doing the work, right? We just ignore errors and log them? It seems okay, but just wondering...

Does this support multidb now, or just ignore databases other than the default?

Is this going to ignore more critical errors and lead to broken dumps?